src/pages/home/slider-home-page/slider-home-page.js

```diff
@@ -1,38 +1,41 @@
 import React from 'react';
 import AwesomeSlider from 'react-awesome-slider';
 import 'react-awesome-slider/dist/styles.css';
-import { useSelector } from 'react-redux';
+import { useStore } from 'react-redux';
```

Comment:
Try to stick with project component structure, use the `useSelector` hook here

Reply:
Fixed